landing/src/lib/components/NeuralTopology.svelte

```
@@ -114,318 +62,1293 @@
+      for (const idx of spineIdx) {
+        const d = Math.abs(nodes[idx].pageFrac - targetFrac);
+        if (d < bestDist) {
+          bestDist = d;
+          best = idx;
+        }
+      }
+      return best;
+    };
+
+    const spineOrder = new Map<number, number>();
+    spineIdx.forEach((idx, i) => spineOrder.set(idx, i));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)